### PR TITLE
telugu translation added

### LINKED ISSUE #13: Make Telugu locale
Use the English (`en.json`) translations from the i18n folder in assets with the exception of the shell folder. Each file name should be named `te.json`
https://github.com/tedaky/memory-game/tree/master/projects/the-application/src/assets/i18n

### PR DESCRIPTION
**Pull request by yakkalipavan**
make telugu locale

Closes #13

### CODE REVIEW

File: projects/the-application/src/assets/i18n/main/te.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "FULL_STOP": ".",
+    "GAME": "ఆట",
+    "HIGH_SCORES": "అధిక స్కోర్లు",
+    "INSTALL": "ఇన్‌స్టాల్ చేయండి",
+    "INSTALL_MEMORY_GAME": "మెమరీ గేమ్‌ను ఇన్‌స్టాల్ చేయండి",
+    "PLEASE_RELOAD": "దయచేసి మళ్లీ లోడ్ చేయండి",
+    "PLEASE_FINISH_OR_RESET_GAME": "దయచేసి ఆట పూర్తి చేయండి లేదా రీసెట్ చేయండి",
+    "RECENT_SCORES": "ఇటీవలి స్కోర్‌లు",
+    "REFRESH": "రిఫ్రెష్ చేయండి",
+    "SETTINGS": "సెట్టింగులు",
```
Comment (tedaky) [inline projects/the-application/src/assets/i18n/main/te.json:14]: Can you add TAP and THEN?

See https://github.com/tedaky/memory-game/pull/16/files#diff-47c31de70082a49e837b88ef47ba61dc365e3d92008c6d09cbacc1a0edbfb98a

**Review by tedaky — CHANGES_REQUESTED** [review]
Please add TAP and THEN to main/te.json

File: projects/the-application/src/assets/i18n/main/te.json (hunk 1 above)
Comment (yakkalipavan) [reply projects/the-application/src/assets/i18n/main/te.json:14]: Added definition for TAP and its good for THEN as well. There is no contextual definition for both TAP and THEN. One definition is enough

**Review by tedaky — APPROVED** [review]
I approve of these translations.
@sandarshsridhar

**Review by sandarshsridhar — APPROVED** [review]

**Comment by tedaky** [comment]
Closes #13

**Pull request merged**